term-utils/wall.c

```diff
@@ -56,11 +56,16 @@
 #include <string.h>
 #include <time.h>
 #include <unistd.h>
-#include <utmpx.h>
 #include <getopt.h>
 #include <sys/types.h>
 #include <grp.h>
 
+#if defined(HAVE_LIBSYSTEMD) && HAVE_DECL_SD_SESSION_GET_USERNAME == 1
```

Comment:
`HAVE_DECL_SD_SESSION_GET_USERNAME` is never defined under meson?

Reply:
I vote for dropping configure.ac and use only meson, meson seems to be much more flexible and better for the checks...

Reply:
Unfortunately meson is
a) still missing obvious features (like man-page translation)
b) silently different from autotools because some defines or checks are missing.

So it's very unlikely that util-linux will drop autotools as primary buildsystem in the near future.
Introducing more checks that don't work under meson also doesn't make that migration easier.